internal/cli/chroot.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Licensed as a Redpanda Enterprise file under the Redpanda Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+// https://github.com/redpanda-data/connect/blob/main/licenses/rcl.md
+
+package cli
+
+import (
+	"syscall"
+)
+
+func chroot(path string) error {
+	if err := syscall.Chdir(path); err != nil {
+		return err
+	}
+	if err := syscall.Chroot(path); err != nil {
+		return err
+	}
+	if err := syscall.Chdir("/"); err != nil {
+		return err
+	}
```

Comment:
Nit: It would be handy to leave a comment explaining this sequence of operations. Not sure why `Chdir` is called twice

Reply:
Removed and added docs.